### DISCUSSION TITLE
Add Toggle for DefaultScene Component for Global or Connection Load.

### DISCUSSION THREAD

**Discussion by Katerlad** [FirstGearGames/FishNet#233] [Feature Request]
While the current setup of DefaultScene component works great for beginners just wanting to get started. It currently forces the Default Online Scene to be Globally Loaded. 

This is purely a QOL suggestion for newer members to have a alternative solution already built for them.

**Requirements:**
- Toggle on the component to load globally or by connection.
- Automatically handle race condition that the PlayerSpawner will spawn the player prefab into the Online Scene and not the Offline Scene.

**WHY?** 
- Global Scenes cannot be unloaded for individual connections, While a Global Default Online scene might work for 50% of the cases, a lot of members may want their default online scene be their OverWorld Scene where they will have connections in and out of it when loading into additive scenes like dungeons or caves.

- So if you wanted a client to load right into the overworld scene upon connection, and then allow connections to then go into additive scenes, you cannot unload a global scene for just one client with the way global scenes work.

**Comment by FirstGearGames** [comment]
Didn't even need to read the why. Great idea!

**Comment by FirstGearGames** [comment]
Added in 3.3.0
